### ISSUE TITLE
Wrong internal help tooltip

### ISSUE THREAD

**Issue by ivanich-fmf** [MightyPirates/OpenComputers#2142]
![2016-11-20_17 03 53](https://cloud.githubusercontent.com/assets/15928505/20463433/bf04a00c-af43-11e6-98a7-9fd57fd17eb3.png)
Minecraft 1.7.10 Forge 10.13.4.1558 OC 1.6.0.9

**Comment by payonel** [comment]
I looked into correcting this and I don't know what these methods should even do; they appear broken at best.

`getItemInventorySize(slot: number)` appears to return the size of a database upgrade at `slot` in the internal inventory. I would rather we call this `getDatabaseSize` unless it works with anything else (i couldn't find what else might be supported)

`dropIntoItemInventory(inventorySlot: number, slot: number[, count: number])` would seem to allow you to set the ghost itemstack in the database upgrade, But it doesn't work - with my tests it was setting all of the slots in the db upgrade  to ghost images of itself and not of the item I had indexed. Before I fix this, I would like to know what the true intent of this method was, and maybe I'm not even using it correctly.

`suckFromItemInventory(inventorySlot: number, slot: number[, count: number])` might be intended to unghost the itemstack metadata from the db upgrade.

I'd like clarification before I fix and document. @Vexatos or @fnuecke ?
